src/components/TitleBody/TitleBody.scss

```
@@ -0,0 +1,3 @@
+.body {
```

we should be a bit overly verbose, body is too generic and could potentially be over written, lets use:
```suggestion
.title-body__body {
```